crate/cli/src/tests/shared/import.rs

```diff
@@ -26,6 +26,7 @@ use crate::{
 
 #[derive(Default, Debug)]
 pub(crate) struct ImportKeyParams {
+    // TODO: should be replaced by ImportKeyAction
```

Comment:
Should we do here, or in the merging branche ?

Reply:
maybe in a another PR if you are OK (because it is not a transparent change, we need to change the function `import_key` signature a little to reuse the `ImportKeyAction` struct. The same for `export_key`.)

Reply:
ok !